lib/streamlit/util.py

```diff
@@ -179,10 +179,10 @@ def calc_md5(s: Union[bytes, str]) -> str:
     return h.hexdigest()
 
 
-def exclude_key_query_params(
-    query_params: Dict[str, List[str]], keys_to_exclude: List[str]
-) -> Dict[str, List[str]]:
-    """Returns new object query_params : Dict[str, List[str]], but without keys defined with keys_to_drop : List[str]."""
+def exclude_keys_in_dict(
```

Comment:
Small refactor as it felt appropriate to change it to not just query_params and I was also facing a typing error so I changed it to `Dict[str, Any]`